src/main/java/seedu/address/logic/commands/edit/DeleteModuleRoleOperation.java

```diff
@@ -37,10 +37,10 @@ protected ModuleRoleMap execute(ModuleRoleMap moduleRoleMapToEdit) throws Comman
         ModuleRoleMap failed = ret.removeAll(moduleRoleMapToDelete);
```

Comment:
I am sorry if I miss anything, but what does `ret` means? I think using short form for variable naming is a bit not very good practice and may be considered as code quality/standard violation? Would it be better to change this to some another more straightforward name? Thank you

Reply:
Sorry for the sloppy naming! This is a bad habit from competitive programming